### ISSUE TITLE
[Request] Provide/Distribute Mullvad VPN through Accrescent Store

### ISSUE THREAD

**Issue by pxlkng** [mullvad/mullvadvpn-app#5606]
Currently the only direct way of getting Mullvad VPN on Android with automatic updates and avoiding Google is through the F-Droid store. This is not optimal and there are several [issues](https://privsec.dev/posts/android/f-droid-security-issues/) with this approach, most notably F-Droids really slow update time, their signing of all apks with their own key and therefore generally weak security model. More reasons are listed in the article.

Probably the best alternative currently available is [Accrescent](https://accrescent.app/), which does the concept of F-Droid right, with strong security mechanics like app signing key pinning, signed repository metadata, unprivileged automatic updates and developers own apk signing keys. (more also on their site)
It also has some other nice features like support for split-apks.

While I don't request that Mullvad stops to distribute Mullvad VPN through F-Droid, **I would like to suggest and strongly recommend also distributing Mullvad VPN through Accrescent** as at least another, safer/better, option and to maybe additionally recommend this way of installing over F-Droid for people that avoid Google Play.

On a more personal note I also feel like Accrescent is much more user-friendly and not as bloated with excess unneeded features like F-Droid. I think it goes well with Mullvads principle of being secure, private and easy to use in one.

Thank you for making Mullvad VPN and I hope that this could be considered.

**Comment by BHydden** [comment]
If this is your first time hearing about Accrescent, please be aware it is still in alpha and lacks many QOL features. It is also white list only currently so if you're interested discuss the matter with the dev otherwise just bookmark it for later when Accrescent is ready for widespread adoption.

**Comment by barbiehunter** [comment]
Are there any news ading Mullvad to Accrescent?

**Comment by Rawa** [comment]
No news as for right now I'm afraid. The feature request is still very much relevant and in our backlog on what we want to do. We've had other projects with more priority to take precedence. I don't want to give out an inaccurate timeline but the teams wish is that we are able to provide this in the near future.

**Comment by Jeffrey2307** [comment]
I would love to see you there too!

**Comment by PrivacyAndSecurity** [comment]
I agree, that we would love to see Mullvad VPN on Accrescent! Maybe you ask why we want this or what Benefits it would have for us as Users.

The Benefits for me would be a few, like these:

- I could get Mullvad VPN from Accrescent, which I already installed and use to download Apps. I prefer Accrescent over Github, because on Accrescent the App get's checked before it comes to the Store and the Apps are always checked, like for example the SDK of Apps, if there is Malware and more.

- It would make it a lot easier for me and save me time, because Mullvad VPN is one of the few Apps, I use in Obtainium to download. And if my last few Apps would come to Accrescent, I wouldn't need to use Obtainium. I just use it until my last few Apps on there come to Accrescent.

- Apps on Github are not checked, but on Accrescent they are going trough Filters always. And I think that would make it more secure. And since Accrescent is on a good way in terms of Security and Privacy, I think that it will be really good in the Future. And I think that Apps which value Security and Privacy should make it available there. Because Accrescent have the same values in Security and Privacy.

- I really hope that you Guys could make that possible, because it would have great Benefits for us as Users. And think like that: If we support them by adding Apps, we help them to grow from Alpha to Stable.

**Comment by PrivacyAndSecurity** [comment]
Is there any Updates on this?

**Comment by Pururun** [comment]
> Is there any Updates on this?

Not much. We want to do this, but we have prioritized other things such as multihop, DAITA and Shadowsocks for example.

**Comment by boredsquirrel** [comment]
Just saying, Multihop seems to just have been added, and DAITA and Shadowsocks are there for a while ;)

**Comment by cameronaaron** [comment]
I would also like to cast a vote for this

**Comment by ghost** [comment]
I would like it to be available on Accrescent as soon as the dev is available to accept app request too

**Comment by albin-mullvad** [comment]
Hi! Here's a small update: We have on our internal roadmap to evaluate and potentially add support for Accrescent later this year. The Android team is quite small and our focus in the build/release area have lately been [reproducible builds](https://mullvad.net/en/blog/reproducible-builds-verify-our-android-app-builds-bit-for-bit) as well as release automation which we've prioritized higher than adding another distribution channel. We also consider it to be more work than just uploading our binaries in another portal. We need to thoroughly evaluate Accrescent as a platform as well as build a robust automated release pipeline before we can start publishing there.

**Comment by git70** [comment]
@ghost
> 1. relying on Appverifier for verification (some apps I use are not in the internal database)
> 2. not having cert pinning and cert transparancy
>
> I know about obtainium and specifically go out of my way to not use it, I only use secure app stores.

1. If the application is not in the internal AppVerifier database, then you can / should check its signature by hand
2. The cert pinning is not made by the store, but the operating system (regardless of the source of the installation). He does it at the first installation, and later during the update he checks if the signature is the same.
3. Opinions on the safety of using stores vs direct download from the developer are divided. For example:
https://discuss.grapheneos.org/d/18731-f-droid-vulnerability-allows-bypassing-certificate-pinning
https://github.com/obfusk/fdroid-fakesigner-poc

@albin-mullvad 
This is a great news that Accrescent is on your road map!
Your explanation is proof that you are professionals and carefully examine every matter before its implementation.

**Comment by ghost** [comment]
Bumping this as well, I'd love to see Mullvad on Accrescent

**Comment by cameronaaron** [comment]
Any kind of update on this?

**Comment by Rawa** [comment]
@cameronaaron 

We've just set the plan for Q4, it includes looking into supporting more stores in October, with possible implementation in November. It is by no means set that we will support Accrescent, we've been hesitant since it is in alpha and the project halted October 2024 to June 2025., however it seems like development has started again. So exactly where this issue heads will be more clear in November.

**Comment by lberrymage** [comment]
> It is by no means set that we will support Accrescent, we've been hesitant since it is in alpha and the project halted October 2024 to June 2025

Accrescent developer here 👋🏻. Just wanted to add some clarification to this: Accrescent hasn't halted. We (Accrescent) said feature development would halt starting November 2025 if we didn't receive enough financial support to continue, but we quickly received enough support to continue through at least June 2026. Development is ongoing.

(edit: June 2026, not 2025)

**Comment by Rawa** [comment]
> > It is by no means set that we will support Accrescent, we've been hesitant since it is in alpha and the project halted October 2024 to June 2025
> 
> Accrescent developer here 👋🏻. Just wanted to add some clarification to this: Accrescent hasn't halted. We (Accrescent) said feature development would halt starting November 2025 if we didn't receive enough financial support to continue, but we quickly received enough support to continue through at least June 2026. Development is ongoing.
> 
> (edit: June 2026, not 2025)
> 

Hello, I think you misunderstood me, I'm talking about 2024 to 2025 June, by looking at git history. This feature request has been existed for about a year now. Im glad to hear you have funding moving forward. 👍 Ill get back and update this issue when we know more.

Best regards
